core/test/docs.toml

```
@@ -16,7 +16,7 @@ time = "basin/time.arrow"
 [allocation]
 timestep = 86400
 use_allocation = true
-objective_type = "quadratic_relative"
+objective_type = "linear_absolute"
```

```suggestion
objective_type = "linear_absolute"  # optional, default "linear_absolute"
```

And could you add such a comment also to `use_allocation` and `timestep`? The latter is optional, but required if `use_allocation`.